wsi_superpixel_guided_labeling/web_client/views/body/ActiveLearningView.js

```diff
@@ -663,19 +664,28 @@ const ActiveLearningView = View.extend({
             restRequest({
                 url: `job/${jobId}`
             }).done((update) => {
-                if (update.status === JobStatus.SUCCESS) {
+                if (update.status >= JobStatus.SUCCESS) {
```

Comment:
We should check explicitly that this is one of `JobStatus.SUCCESS`, `JobStatus.FAILED`, `JobStatus.CANCELED`.  I know there were some other states added by plugins that are higher numbered and shouldn't be considered done.